bmi_topography/cli.py

```diff
@@ -73,9 +73,12 @@ def main(quiet, dem_type, south, north, west, east, output_format, api_key, no_f
     topo = Topography(dem_type, south, north, west, east, output_format)
     if not no_fetch:
         if not quiet:
-            click.secho("Fetching data...", fg="yellow")
-        topo.fetch()
+            click.secho("Fetching data...", fg="yellow", err=True)
+        path_to_dem = topo.fetch()
         if not quiet:
             click.secho(
-                "File downloaded to {}".format(getattr(topo, "cache_dir")), fg="green"
+                "File downloaded to {}".format(getattr(topo, "cache_dir")),
+                fg="green",
+                err=True,
             )
+        print(path_to_dem)
```

Comment:
Should this be quiet, as well?

Reply:
I don't think so since it's the main output of the command. If a user really would like to silence it the could do,
```bash
$ bmi-topography ... > /dev/null
```

Reply:
👍